.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+	"parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+	},
+	"rules": {
+		"indent": ["error", 2, { "VariableDeclarator": { "var": 2, "let": 2, "const": 3 } }],
```

Comment:
Weird spacing?

Reply:
This is a JSON file by the looks of it. Maybe that's why the spacing is weird

Reply:
Looked at file, we do have mixed tabs and spaces. Need to fix that.

Reply:
so i just need to add a .json extension?

Reply:
Need to fix formatting, and add .json extension. whenever you mix tabs and spaces in the same line to indent, some machines, editors will recognize them differently and can cause conflict

Reply:
just to clarify, you only want to be able to use tab when you're indenting right? is there any other time we would use tabs?

Reply:
I can't think of any time we use the tab key for anything else @jwu910 @phuchle @AlChiu

Reply:
@rainrivas @psyduc I think the goal here is to make the indent length equal to 2 spaces.  In the editor, pressing the tab button should translate to 2 spaces.